detection_rules/etc/packages.yml

```diff
@@ -4,27 +4,27 @@ package:
     maturity:
     - production
   log_deprecated: true
-  name: '8.13'
+  name: '8.14'
   registry_data:
     categories:
     - security
     conditions:
-      kibana.version: ^8.13.0
       elastic:
         subscription: basic
+      kibana.version: ^8.14.0
     description: Prebuilt detection rules for Elastic Security
     format_version: 3.0.0
     icons:
     - size: 16x16
       src: /img/security-logo-color-64px.svg
       type: image/svg+xml
-    source:
-      license: Elastic-2.0
```

Comment:
@shashank-elastic - this should not have been removed. May be a bug in the CLI command.